config/config.go

```diff
@@ -51,9 +51,9 @@ type ForwardTo struct {
 	DimensionsOrder      []string                    `json:",omitempty"`
 	Filters              *filtering.FilterObj        `json:",omitempty"`
 	TraceSample          *sampling.SmartSampleConfig `json:",omitempty"`
+	TraceDistributor     *sampling.SmartSampleConfig `json:",omitempty"`
```

Comment:
I understand why it ends up being done this way, but it means there is probably nothing preventing people from putting sampler config things (like `BackupLocation` or `ClusterPercent`) in their distributor config, which can be very confusing.

Reply:
Even if i created a separate object, they could put in a key of `BackupLocation` and we would ignore it just as we'll currently ignore it. This solution i have put here makes the backend to the sampler far simpler codewise.